indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -324,11 +323,13 @@
 (defn bulk-index-autocomplete-suggestions
   "Save a batch of suggestion documents in Elasticsearch."
   [context docs]
-  (doseq [docs-batch (partition-all MAX_BULK_OPERATIONS_PER_REQUEST docs)]
-    (let [bulk-operations (cmr-bulk/create-bulk-index-operations docs-batch)
-          conn (context->conn context)
-          response (bulk/bulk conn bulk-operations)]
-      (handle-bulk-index-response response))))
+  (if (esi/index-exists? (context->conn context) "1_autocomplete")
```

Comment:
I don't think this check is necessary. The `1_autocomplete` index should be created as part of the `db-migration` like all the other indices used by CMR. If `1_autocomplete` index is not automatically initialized for any reason, we should fix that in db-migration. It is wrong to check if the index exists in every single bulk index request.